main/webapp/modules/core/scripts/reconciliation/standard-service-panel.js

```diff
@@ -98,7 +99,10 @@ ReconStandardServicePanel.prototype._constructUI = function() {
   this._elmts.typeInput.attr('aria-label',$.i18n('core-recon/type'))
 
   this._elmts.rawServiceLink.attr("href", this._service.url);
-
+  fetch(this._service.url)
+  .then(response => response.json())
+  .then(apiData => this._elmts.documentationLink.attr("href",apiData.documentation));
```

Comment:
This would throw an error and crash if used on a service without a "documentation" property.